samples/excel/89-preview-apis/events-tablecollection-changed.yaml

```diff
@@ -0,0 +1,164 @@
+order: 9
+id: excel-events-tablecollection-changed
+name: Events - Table collection changed
+description: Add event handlers for table collection onDataChanged event
+host: EXCEL
+api_set:
+    ExcelApi: 1.7
+script:
+    content: |+
+        $("#setup").click(() => tryCatch(setup));
+        $("#register-on-data-changed-handler").click(() => tryCatch(registerOnDataChangedHandler));
+        $("#change-data").click(() => tryCatch(changeData));
+
+        async function registerOnDataChangedHandler() {
+            await Excel.run(async (context) => {
+                let table = context.workbook.tables;
```

Comment:
Suggest that you change this variable name to "tableCollection" (instead of "table").